sensing/autoware_pointcloud_preprocessor/src/filter.cpp

```diff
@@ -70,15 +70,15 @@ autoware::pointcloud_preprocessor::Filter::Filter(
 {
   // Set parameters (moved from NodeletLazy onInit)
   {
-    tf_input_frame_ = static_cast<std::string>(declare_parameter("input_frame", ""));
-    tf_output_frame_ = static_cast<std::string>(declare_parameter("output_frame", ""));
+    tf_input_frame_ = declare_parameter<std::string>("input_frame");
+    tf_output_frame_ = declare_parameter<std::string>("output_frame");
+    max_queue_size_ = static_cast<size_t>(declare_parameter<int64_t>("max_queue_size"));
     has_static_tf_only_ = static_cast<bool>(declare_parameter("has_static_tf_only", false));
```

Comment:
This seems to be using the old way of declaring parameters

Reply:
*note: I am referring to `has_static_tf_only_`

Reply:
@amadeuszsz will remove the default parameters for `has_static_tf_only_` for all of the nodes after schema-related PRs are merged :)

Reply:
@vividf 
Sure, but if that does not happen in the near future, please send a PR. It will look weird otherwise

Reply:
(I will be mark as approved but leave the conversation open for future reference)